### ISSUE TITLE
Error on Paper 1.21 on player connect

### ISSUE THREAD

**Issue by SlimeDog** [Mr-EmPee/MysticalBarriers#18]
Paper 1.21-127
MysticalBarriers 2.3.0

On player connect
```
[13:11:21 ERROR]: Could not pass event AsyncPlayerSendCommandsEvent to MysticalBarriers v2.3.0
java.lang.NoSuchMethodError: 'io.github.mr-empee.mysticalbarriers.com.mojang.brigadier.tree.RootCommandNode com.destroystokyo.paper.event.brigadier.AsyncPlayerSendCommandsEvent.getCommandNode()'
        at MysticalBarriers-2.3.0.jar/io.github.mr-empee.mysticalbarriers.me.lucko.commodore.PaperCommodore.onPlayerSendCommandsEvent(PaperCommodore.java:102) ~[MysticalBarriers-2.3.0.jar:?]
        at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:40) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:1.21-127-4e6a2a1]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.21.jar:1.21-127-4e6a2a1]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[paper-1.21.jar:1.21-127-4e6a2a1]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.event.Event.callEvent(Event.java:45) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.commands.Commands.sendAsync(Commands.java:509) ~[paper-1.21.jar:1.21-127-4e6a2a1]
        at net.minecraft.commands.Commands.lambda$sendCommands$6(Commands.java:475) ~[paper-1.21.jar:1.21-127-4e6a2a1]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
        at java.base/java.lang.Thread.run(Thread.java:1570) ~[?:?]
[13:11:21 ERROR]: Could not pass event AsyncPlayerSendCommandsEvent to MysticalBarriers v2.3.0
java.lang.NoSuchMethodError: 'io.github.mr-empee.mysticalbarriers.com.mojang.brigadier.tree.RootCommandNode com.destroystokyo.paper.event.brigadier.AsyncPlayerSendCommandsEvent.getCommandNode()'
        at MysticalBarriers-2.3.0.jar/io.github.mr-empee.mysticalbarriers.me.lucko.commodore.PaperCommodore.onPlayerSendCommandsEvent(PaperCommodore.java:102) ~[MysticalBarriers-2.3.0.jar:?]
        at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:40) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:1.21-127-4e6a2a1]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.21.jar:1.21-127-4e6a2a1]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[paper-1.21.jar:1.21-127-4e6a2a1]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.event.Event.callEvent(Event.java:45) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.commands.Commands.sendAsync(Commands.java:509) ~[paper-1.21.jar:1.21-127-4e6a2a1]
        at net.minecraft.commands.Commands.lambda$sendCommands$6(Commands.java:475) ~[paper-1.21.jar:1.21-127-4e6a2a1]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
        at java.base/java.lang.Thread.run(Thread.java:1570) ~[?:?]
```

**Comment by Mr-EmPee** [comment]
Thanks, fixed by https://github.com/Mr-EmPee/MysticalBarriers/commit/4d6f9e32b722a210e3b268daf864476fd2512423

Available new release on spigot

**Comment by SlimeDog** [comment]
Confirmed fix with MysticalBarriers 2.3.1 on Paper 1.21-127. Thanks for fast response.
